webextensions/sidebar/components/TabLabelElement.js

```diff
@@ -99,7 +105,9 @@ export class TabLabelElement extends HTMLElement {
   }
 
   get _content() {
-    return this.querySelector(`.${kCONTENT_CLASS_NAME}`);
+    const shadow = this.shadowRoot;
```

Comment:
<code>return this.shadowRoot.querySelector(`[part=${kPART_TEXT_SPAN}]`)</code> looks enough short here.